api/api.spec.yaml

```
@@ -334,6 +339,9 @@ definitions:
       account:
         title: Account
         type: string
+      org_id:
+        title: Org ID
```

\<nitpick\/\>
```suggestion
        title: Org ID
        description: Identifies the organization that the given resource belongs to
```

as per https://docs.google.com/document/d/19dEXx087osJ8cNmbY5LvDEZKGDytjY9RifJBIP5mME4/edit#heading=h.5e5139cc3hwy

---

nitpicks are good :)